_posts/2017-11-02-totw-107.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Tip of the Week #107: Reference Lifetime Extension"
+layout: tips
+sidenav: side-nav-tips.html
+published: draft
+permalink: tips/107
+type: markdown
+---
+
+# Tip of the Week #107: Reference Lifetime Extension
+
+Originally posted as totw/107 on 2015-12-10
+
+By Titus Winters (titus@google.com)
+
```

Comment:
Add:

"Updated 2017-11-02"

Reply:
It isn't meaningfully updated (so I wasn't making a note of it).